src/components/Api.js

```diff
@@ -0,0 +1,204 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    // baseUrl: "https://around-api.en.tripleten-services.com/v1"
+    this._baseUrl = baseUrl;
+    this._headers = headers;
+  }
+
+  /*---------------------------------------------------*/
+  /*         Profile Picture Api functions             */
+  /*---------------------------------------------------*/
+
+  // – Update avatar
+  // PATCH https://around-api.en.tripleten-services.com/v1/users/me/avatar
+  patchProfileAvatar(link) {
+    return fetch(this._baseUrl + "/users/me/avatar", {
+      method: "PATCH",
+      headers: {
+        authorization: this._headers.authorization,
+        "Content-Type": "application/json",
+      },
```

Comment:
According to the task you need to pass full `headers` into the constructor of `Api`. See here https://snipboard.io/pRdaAZ.jpg . And in the code you need to use only: `headers: this._headers` (not to duplicate the code)